<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commAudioClass</class>
 <widget class="QMainWindow" name="commAudioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>commAudio</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="song_list_on_server">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>511</width>
      <height>181</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Art</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Song</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Artist</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Album</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Year</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Player</string>
    </property>
    <widget class="QSlider" name="player_volume">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="player_audio_duration">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>58</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2:40</string>
     </property>
    </widget>
    <widget class="QSlider" name="player_seekbar">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>68</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGraphicsView" name="player_audio_pic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>28</y>
       <width>81</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="player_audio_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Song name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="player_play_pause_toggle_btn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Resources/play.png</normaloff>Resources/play.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="player_next_btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Resources/next.png</normaloff>Resources/next.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="player_previous_btn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Resources/previous.png</normaloff>Resources/previous.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Resources/volume.png</normaloff>Resources/volume.png</iconset>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_save_current_song">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save this song</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>20</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Song List:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chat:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="chat_history">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>201</width>
      <height>301</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="chat_message">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send_chat_message_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/chat.png</normaloff>Resources/chat.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="push_to_talk_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Push To Talk</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/microphone.png</normaloff>Resources/microphone.png</iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="playlist_client">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Art</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Song</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Artist</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Album</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Year</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Playlist:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect_to_Server"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Playlist"/>
    <addaction name="actionUpload_a_Song"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Playlist"/>
    <addaction name="actionSave_Current_Song"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_Guide"/>
    <addaction name="actionProgram_Documentation"/>
    <addaction name="actionAbout_Program"/>
    <addaction name="actionAbout_Authors"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QMenu" name="menuRadio">
     <property name="title">
      <string>Radio</string>
     </property>
     <addaction name="actionNormal_Radio"/>
     <addaction name="actionMulticast_Radio_2"/>
    </widget>
    <addaction name="actionNormal_Playlist_Mode"/>
    <addaction name="separator"/>
    <addaction name="menuRadio"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUser_Guide">
   <property name="text">
    <string>User Guide</string>
   </property>
  </action>
  <action name="actionProgram_Documentation">
   <property name="text">
    <string>Program Documentation</string>
   </property>
  </action>
  <action name="actionAbout_Program">
   <property name="text">
    <string>About Program</string>
   </property>
  </action>
  <action name="actionAbout_Authors">
   <property name="text">
    <string>About Authors</string>
   </property>
  </action>
  <action name="actionConnect_to_Server">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionMulticast">
   <property name="text">
    <string>Multicast</string>
   </property>
  </action>
  <action name="actionSave_Playlist">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Playlist</string>
   </property>
  </action>
  <action name="actionSave_Current_Song">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Current Song</string>
   </property>
  </action>
  <action name="actionOpen_Playlist">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Playlist</string>
   </property>
  </action>
  <action name="actionUpload_a_Song">
   <property name="text">
    <string>Upload a Song</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNormal_Playlist_Mode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normal Mode</string>
   </property>
  </action>
  <action name="actionMulticast_Radio">
   <property name="text">
    <string>Multicast Radio</string>
   </property>
  </action>
  <action name="actionNormal_Radio">
   <property name="text">
    <string>Normal Radio</string>
   </property>
  </action>
  <action name="actionMulticast_Radio_2">
   <property name="text">
    <string>Multicast Radio</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="commaudio.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionConnect_to_Server</sender>
   <signal>triggered()</signal>
   <receiver>commAudioClass</receiver>
   <slot>newConnectDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpload_a_Song</sender>
   <signal>triggered()</signal>
   <receiver>commAudioClass</receiver>
   <slot>newAudioUploadDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>commAudioClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player_play_pause_toggle_btn</sender>
   <signal>clicked()</signal>
   <receiver>commAudioClass</receiver>
   <slot>playPauseButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>send_chat_message_btn</sender>
   <signal>clicked()</signal>
   <receiver>commAudioClass</receiver>
   <slot>sendMessageButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push_to_talk_btn</sender>
   <signal>pressed()</signal>
   <receiver>commAudioClass</receiver>
   <slot>pushToTalkButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push_to_talk_btn</sender>
   <signal>released()</signal>
   <receiver>commAudioClass</receiver>
   <slot>pushToTalkButtonReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_save_current_song</sender>
   <signal>stateChanged(int)</signal>
   <receiver>commAudioClass</receiver>
   <slot>saveToFileCheckboxHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newConnectDialog()</slot>
  <slot>newAudioUploadDialog()</slot>
  <slot>playPauseButtonClick()</slot>
  <slot>sendMessageButtonClick()</slot>
  <slot>pushToTalkButtonPressed()</slot>
  <slot>pushToTalkButtonReleased()</slot>
  <slot>saveToFileCheckboxHandler()</slot>
 </slots>
</ui>
